docs/reference/pages/panels.rst

```diff
@@ -243,6 +243,24 @@ SnippetChooserPanel
     See :ref:`snippets` for more information.
 
 
+HelpPanel
+---------
+
+.. module:: wagtail.admin.edit_handlers
+
+.. class:: HelpPanel(content, template)
```

Comment:
Please use exactly the same definition as in the code. The default values are missing, as well as the arguments `heading` and `classname`.